### ISSUE TITLE
FR: Additional Highlighting for $INCLUDE, $MetaCommands, and OPTION

### ISSUE THREAD

**Issue by grymmjack** [QB64Official/vscode#121]
I would love it if there was additional highlighting for:

`'$INCLUDE:`

`'$*` Meta-commands ($IF, $LET, $ENDIF, etc.)

`OPTION`

**Comment by LordDurus** [comment]
Option is already 'blue'  do you want it include in the metacommands scope?
![image](https://github.com/QB64Official/vscode/assets/2650021/faab472f-ba4c-4959-a490-a14c90c6ed6c)

**Comment by grymmjack** [comment]
Hi @LordDurus been a while :)

Is `OPTION` a meta command? I thought they all started with `$whatever` ? hrmm.

If it's a meta command I guess make it part of that group? 

🙏🏻

**Comment by LordDurus** [comment]
I didn't change $Include, that is a bigger ball of wax since it will have go to definition side effects.
If include is a must add an item just for it.

I don't know if "Option Explicit" is technically a meta command but I do like it having colored as one.  So I included it :)

I did add a scope metacommand.QB64 and it includes
REM $DYNAMIC
' $DYNAMIC
REM $STATIC
' $STATIC
$ASSERTS
$CHECKING
$COLOR
$CONSOLE
$DEBUG
$ERROR
$EXEICON
$LET
$IF
$ELSEIF
$END IF
$SCREENHIDE
$SCREENSHOW
$VIRTUALKEYBOARD
$VERSIONINFO:Comments
$VERSIONINFO:CompanyName
$VERSIONINFO:FileDescription
$VERSIONINFO:FileVersion
$VERSIONINFO:InternalName
$VERSIONINFO:LegalCopyright
$VERSIONINFO:LegalTrademarks
$VERSIONINFO:OriginalFilename
$VERSIONINFO:ProductName
$VERSIONINFO:ProductVersion
$VERSIONINFO:Web
Option Explicit

This is in dev v0.9.1
